.github/workflows/release.yml

```diff
@@ -0,0 +1,124 @@
+name: Publish package to GitHub Packages
+on:
+  release:
+    types: [published]
+    
+env:
+  REGISTRY: ghcr.io
+  IMAGE_NAME: IsantePlus/docker-isanteplus-server
+  ISANTEPLUS_VERSION: 2.9.1
+  
+jobs:
+  publish-package:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-java@v1
+        with:
+          java-version: 8.0.232
+          server-id: github-packages # Value of the distributionManagement/repository/id field of the pom.xml
+          server-username: pmanko
+          server-password: ${{ secrets.TOKEN }} # env variable for token in deploy
+      - uses: s4u/maven-settings-action@v2.2.0
+        with:
+          servers: |
+            [{
+                "id": "github-packages",
+                "username": "${{ secrets.RELEASE_USERNAME }}",
+                "password": "${{ secrets.RELEASE_TOKEN }}"
+            }]
+          githubServer: false
+      - name: Publish package
+        run: mvn -DskipTests -B -U deploy
+        env:
+          GITHUB_TOKEN: ${{ secrets.RELEASE_TOKEN }}
+           
+  Docker-build:
+    name: Push Docker image to GitHub Packages
+    runs-on: ubuntu-latest
+    env:
+      USERNAME: ${{ secrets.RELEASE_USERNAME }}
+      TOKEN: ${{ secrets.RELEASE_TOKEN }}
+    steps:
+      - uses: actions/checkout@v2
+      - name: Get latest Released Tag
+        uses: actions-ecosystem/action-get-latest-tag@v1
+        id: get-latest-tag
+      - name: Test Tag
+        run: |
+           echo ${{ steps.get-latest-tag.outputs.tag }} 
```

Comment:
Hey @mozzy11  ,was this commit supposed to be for openmrs or isante distribution